### DISCUSSION TITLE
Is there an initiate only command?

### DISCUSSION THREAD

**Discussion by adaliszk** [PurpurMC/Purpur#936] [__Dismissed__] [answered]
Hey,

is there a way to run the jar file without actually starting a server but initialising the folders and configuration files?

I wish to wrap up the needed dependencies into a docker container so that when I would run the server it can already boot and generate the world.

Best,
Ádám

**Comment by katekerllenevich** [comment, marked as answer]
Alas there is not. Best bet is to start the server then delete worlds and such. Then you have the files you need.
